tests/Unit/Core/Image/Parser/ImageTagSourceParserTest.php

```diff
@@ -38,7 +38,7 @@ class ImageTagSourceParserTest extends TestCase
 
     public function setUp()
     {
-        $this->parser = new ImageTagSourceParser('/my-shop');
```

Comment:
Does `$this->parser = new ImageTagSourceParser('/my-shop');` works too ? Can we test for both cases ?

Reply:
i guess we could, this _may_ require to update service itself, but we dont need that use case, at least for now. :)

Reply:
What I mean is you updated the test to use `/my-shop/` instead of `/my-shop`

So previous implementation worked without the trailing `/`, new implementation works without. Can we make an implementation that works for both ?

Reply:
if you really want to do that (even though we dont have a use case for that atm), ill make separate PR for it. :)